src/framework/mlt_repository.c

```diff
@@ -98,12 +98,51 @@ mlt_repository mlt_repository_init( const char *directory )
 	free(newpath);
 #endif
 
+	char **denylist  = NULL;
+	int dl_length = 0;
+	if (getenv( "MLT_REPOSITORY_DENY" ))
+	{
+		char *p = strtok (getenv( "MLT_REPOSITORY_DENY" ), ":");
```

Comment:
How about using `mlt_tokeniser_parse_new()` and related?

Reply:
Wasn't aware of this, done.